docs-ref-conceptual/zone-pivot-groups.yml

```diff
@@ -8,6 +8,8 @@ groups:
       title: apt (Ubuntu, Debian)
     - id: dnf
       title: dnf (RHEL, Fedora, CentOS)
+    - id: tdnf
+      title: tdnf (Mariner)
```

Comment:
@jiasli , YML, so I don't know why we even have a JSON.  I'm checking with @DavidCBerry13  to see why he added both.

Reply:
@jiasli, here's the scoop ... Our repo, for unknown reason, is not handling the traditional zone pivot setup.  In this screen print, you'll see a reference on line 12 ...
![image](https://user-images.githubusercontent.com/58762114/166070605-c38a5f62-894f-4cc5-8510-1909dea6ab53.png)

... this is because this is actually an internal JSON file that gets built during the publishing process sourcing from the YML file.  The JSON file of the same name, checked into GitHub, is not in use and @DavidCBerry13, who originally setup our zone pivot repo files, is deleting it.

**ACTION ITEM:**  Remove the docs-ref-conceptual/zone-pivot-groups.json file from this PR.